.buildkite/scripts/pipelines/pull_request/pipeline.ts

```diff
@@ -176,7 +176,9 @@ const uploadPipeline = (pipelineContent: string | object) => {
       GITHUB_PR_LABELS.includes('ci:cypress-burn') ||
       GITHUB_PR_LABELS.includes('ci:all-cypress-suites')
     ) {
-      pipeline.push(getPipeline('.buildkite/pipelines/pull_request/cypress_burn.yml'));
+      pipeline.push(
+        getPipeline('.buildkite/pipelines/pull_request/security_solution/cypress_burn.yml')
+      );
```

Comment:
That's probably out of the scope of this PR, but I'd like us to consider running `cypress_burn` tests not only when developers want them to run by adding labels (`ci:cypress-burn` or `ci:all-cypress-suites`), but also along with our normal tests that run when there are changes on the file system.

Reply:
We'll follow-up on this, thanks! :)